test/cli/commands/test_export_game.py

```diff
@@ -1,15 +1,23 @@
 from pathlib import Path
 from unittest.mock import MagicMock
 
+import pytest
 from caver.patcher import CSPlatform
 
 from randovania.cli.commands import export_game
 from randovania.games.cave_story.exporter.game_exporter import CSGameExportParams
 from randovania.games.dread.exporter.game_exporter import DreadGameExportParams, DreadModPlatform
 from randovania.games.game import RandovaniaGame
 
+game_ignore_list = [
+    RandovaniaGame.FUSION,
+]
+
 
 def test_add_parser_arguments_for(game_enum: RandovaniaGame) -> None:
+    if game_enum in game_ignore_list:
+        pytest.skip(f"{game_enum} is ignored for this test.")
```

Comment:
This is not necessary. All the test does is get export params. What's happening is that Fusion doesn't have the new thing I added to all games

Reply:
Do I just have to align fusions exporter with the similar changes in https://github.com/randovania/randovania/pull/6102/files#diff-ce2e449c3b38198543a262f2088cc209f644e2a48c85ac7fa403f976352cd7ec

Reply:
https://github.com/randovania/randovania/pull/6086/files actually

Reply:
thanks! done.